### DISCUSSION TITLE
Per post `showReadingTime` param?

### DISCUSSION THREAD

**Discussion by budimanjojo** [Lednerb/bilberry-hugo-theme#430] [Ideas]
Hello, I want `showReadingTime` is true by default, but I don't want all pages to show reading time. Is there a reason to not letting the post to override this param?
https://github.com/Lednerb/bilberry-hugo-theme/blob/31404c6404d5dc674ff93869f77191588d947677/layouts/partials/default-content.html#L17
Maybe something like this?
```
{{ if ( .ctx.Params.showReadingTime | default .ctx.Site.Params.showReadingTime | default false ) }}
```
Thanks before.

**Comment by budimanjojo** [comment]
Sorry about the code, maybe u guys have a better knowledge on how to implement the code, because my code will not work in situation where the user set global `showReadingTime` to `false` but want to override the post param to `true`.

**Comment by budimanjojo** [comment]
should I create a PR?

**Reply by igor-baiborodine** [reply]
@budimanjojo, let me check first.

**Comment by igor-baiborodine** [comment]
@budimanjojo, now this feature is available with the release of `v3.13.0`.

![image](https://user-images.githubusercontent.com/1027701/185650952-12b5ba1a-206f-4259-9c05-893bbb3e6d7a.png)
